.pre-commit-config.yaml

```diff
@@ -5,7 +5,7 @@ repos:
       - id: end-of-file-fixer
         exclude: ^internal/(?:go|pprof)/
       - id: trailing-whitespace
-        exclude: ^internal/(?:go|pprof)/
+        exclude: ^internal/(?:go|pprof)/|^tests/testdata/
```

Comment:
@maxbrunet Hi :wave: Any idea why the pre-commit-lite bot is not taking the changes of .pre-commit-config into the account? I thought these changes combined with the ones in .golangci should make the pre-commit-lite bot from making any changes in the PR. What am I doing wrong?

Reply:
I had also tried `paths_ignore` from here yesterday: https://github.com/marketplace/actions/skip-duplicate-actions#skip-ignored-paths but that didn't seem to work either :(